### ISSUE TITLE
ONNX Conversion

### ISSUE THREAD

**Issue by JohnnyOpcode** [mozilla/DeepSpeech#3659]
I tried to convert the 0.9.3 model into ONNX, but received the following errors.

2021-06-15 13:21:21,385 - INFO - Using tensorflow=2.3.0, onnx=1.9.0, tf2onnx=1.8.5/50049d
2021-06-15 13:21:21,385 - INFO - Using opset <onnx, 13>
2021-06-15 13:21:21,864 - ERROR - Tensorflow op [AudioSpectrogram: TFL_AudioSpectrogram] is not supported
2021-06-15 13:21:21,864 - ERROR - Tensorflow op [Mfcc: TFL_Mfcc] is not supported
2021-06-15 13:21:21,864 - ERROR - Unsupported ops: Counter({'TFL_AudioSpectrogram': 1, 'TFL_Mfcc': 1})
2021-06-15 13:21:21,875 - INFO - Optimizing ONNX model
2021-06-15 13:21:23,642 - INFO - After optimization: Cast -3 (3->0), Const -109 (174->65), Identity -18 (18->0), Transpose -21 (21->0)
2021-06-15 13:21:23,802 - INFO - 
2021-06-15 13:21:23,802 - INFO - Successfully converted TensorFlow model deepspeech-0.9.3-models.tflite to ONNX
2021-06-15 13:21:23,802 - INFO - Model inputs: ['input_node', 'previous_state_c', 'previous_state_h', 'input_samples']
2021-06-15 13:21:23,803 - INFO - Model outputs: ['logits', 'new_state_c', 'new_state_h', 'mfccs', 'metadata_version', 'metadata_sample_rate', 'metadata_feature_win_len', 'metadata_feature_win_step', 'metadata_beam_width', 'metadata_alphabet', 'metadata_language']
2021-06-15 13:21:23,803 - INFO - ONNX model is saved at deepspeech-0.9.3.onnx

I used this command

python -m tf2onnx.convert --opset 13 --tflite deepspeech-0.9.3-models.tflite --output deepspeech-0.9.3.onnx

My question is how are these Tensorflow Ops encoded/fused to work in TFlite?

Thx.
Johnny

**Comment by lissyx** [comment]
Please reach for support on Discourse.

**Comment by lissyx** [comment]
`AudioSpectrogram` and `MFCC` are manually exposed  to the TFLite runtime on our tensorflow fork, as much as I remember.
